### PR TITLE
Add CLI commands reference page

### PR DESCRIPTION
**Pull request by dotNetkow**
The Capacitor CLI commands are sprinkled throughout the docs, but there's no main reference list like with the Ionic CLI.

This PR adds a dedicated page under the Basics section. Manually created for now, in the future we could automate it with a template.

I added helpful usage details - please call out any additional details worth adding, if any.

Played around with various formatting - I think what I landed on looks clean, easy to read, etc.

Direct page here: https://capacitor-site-git-cli-commands.ionic1.vercel.app/docs/basics/cli-commands

### CODE REVIEW

File: docs-md/basics/cli-commands.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+<p class="intro">The Capacitor command-line interface (CLI) tool is used to develop Capacitor apps.</p>
+
+## Installation
+
+Capacitor does not use a global CLI. Instead, CLI is installed locally into each project as an npm script. This makes it easier to manage versions of Capacitor across many different apps.
+
+```bash
+cd my-app
+npm install @capacitor/core @capacitor/cli
+```
```
Comment (dwieeb) [inline docs-md/basics/cli-commands.md:20]: Is this necessary or can we link to the installation page?

File: docs-md/basics/cli-commands.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+## Add
+
+Add a native platform project to your project.
+
+```bash
+npx cap add [platform]
```
Comment (dwieeb) [inline docs-md/basics/cli-commands.md:27]:
```suggestion
npx cap add <platform>
```

Required arguments use `<` and `>`. Optional is `[` and `]`.

File: docs-md/basics/cli-commands.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+npx @capacitor/cli create [options] [directory] [name] [id]
+```
+
+<strong>Inputs:</strong>
+- `directory` (optional): Directory to create the new app in, such as `c:\src\myapp` 
+- `name` (optional): App name
+- `id` (optional): App Package Id (in Java package format, no dashes), such as `com.example.app`
+
+<strong>Options:</strong>
+- `--npm-client [npmClient]`: npm client to use for dependency installation
```
Comment (dwieeb) [inline docs-md/basics/cli-commands.md:80]:
```suggestion
- `--npm-client <npmClient>`: npm client to use for dependency installation
```

File: docs-md/basics/cli-commands.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+npx cap init [options] [appName] [appId]
+```
+
+<strong>Inputs:</strong>
+- `appName` (optional): Name of app
+- `appId` (optional): App Package Id (in Java package format, no dashes), such as `com.example.app`
+
+<strong>Options:</strong>
+ - `--web-dir [value]`: Directory of your project's built web assets (default: `www`)
+ - `--npm-client [npmClient]`: npm client to use for dependency installation
```
Comment (dwieeb) [inline docs-md/basics/cli-commands.md:122]:
```suggestion
 - `--web-dir <value>`: Directory of your project's built web assets (default: `www`)
 - `--npm-client <npmClient>`: npm client to use for dependency installation
```

File: docs-md/basics/cli-commands.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+    cordova-plugin-camera
+    cordova-plugin-splashscreen
+```
+
+## Open
+
+Opens the native project workspace in the specified native IDE (Xcode for iOS, Android Studio for Android). Once open, use the native IDEs to build, simulate, and run your app on a device.
+
+```bash
+npx cap open [platform]
```
Comment (dwieeb) [inline docs-md/basics/cli-commands.md:149]:
```suggestion
npx cap open <platform>
```

File: docs-md/basics/cli-commands.md (hunk 1 above)
Comment (dotNetkow) [reply docs-md/basics/cli-commands.md]: true... removed the heading but mention it in the beginning sentence.

**Pull request merged**